commands/en/tos.js

```diff
@@ -3,10 +3,29 @@
 module.exports = {
   name: 'tos',
   category: 'utility',
-  description: '[Mandarin Only] Terms Of Services',
+  description: 'Terms of Service for this bot, please be sure to abide by it.',
   aliases: [],
   run: async (bot, msg, args) => {
-    // 如果不想顯示該指令 請刪掉該檔案 owo
-    return msg.channel.send(':x: **Sorry, the command hasn\'t been translated yet!**')
+    // Delete this file if you dont want this command ;w;
+    const { config, MessageEmbed } = bot
+    return msg.channel.send({
+      embeds:
+        [
+          new MessageEmbed()
+            .setTitle('v6 Music Bot | Terms of Service')
+            .setColor('FF2323')
+            .setDescription("The words of which the initial letter is capitalized have meanings defined under the following conditions.\nThe following definitions shall have the same meaning regardless of whether they appear in singular or in plural.")
+            .addField("Definitions:", "Application\nmeans the software program provided by the NCT-skyouo downloaded by You on any electronic device, named V6 Music Bot\n\nYou\nmeans the individual accessing or using the service, or the company, or other legal entity on behalf of which such individual is accessing or using the service, as applicable.")
+            .addField("1.", "In order to comply with Article 55 of 'Fair Usage Policy(合理使用原則)' of the Copyright Law of the Republic of China(中華民國著作權法),\nYou may not use the public part of this bot for commercial purposes,\nIf you do not live in the legal territory of the Republic of China, the above rules are still valid.")
+            .addField("2.", "You may not use the service to do anything expressly prohibited in Discord ToS/Guideline.")
+            .addField("3.", "You may not reproduce, adapt, distribute, or relicense Kristen v6 unless you obtain the source code of Kristen v6 in a formal manner.")
```

Comment:
Recommend: replace relicense with sublicense
```suggestion
            .addField("3.", "You may not reproduce, adapt, distribute, or sublicense Kristen v6 unless you obtain the source code of Kristen v6 in a formal manner.")
```